src/Host/Broker/Impl/Sessions/Session.cs

```diff
@@ -83,11 +83,12 @@ public class Session {
 
             var useridentity = User as WindowsIdentity;
             // In remote broker User Identity type is always WindowsIdentity
-            string suppressUI = (useridentity == null) ? string.Empty : "--suppress-ui ";
+            string suppressUI = (useridentity == null) ? string.Empty : "--rhost-suppress-ui ";
+            string isRepl = Id.StartsWithIgnoreCase("REPL") ? string.Empty : "--rhost-interactive ";
```

Comment:
@AlexanderSher Is this the best way to flow this info through? I think we might want a new boolean flag here as well, instead of checking names.